### DISCUSSION TITLE
Enabling CORS does not apply to the stream URL

### DISCUSSION THREAD

**Discussion by 5nyper** [AxisCommunications/discussions#231] [VAPIX]
I have updated the headers in the http response of the axis camera as per the instructions here: https://www.axis.com/vapix-library/subjects/t10175981/section/t10154925/display

Like so:
![fdc20d7a-c997-4a8c-b406-51efcb28a040](https://user-images.githubusercontent.com/9522710/234403738-6403d7c6-1a4f-460a-b3e7-381667a6f86d.jpg)

But when I try to access the stream URL (http://<IP_ADDR>/axis-cgi/mjpg/video.cgi) the headers are not updated.

Did i go about this correctly? My goal is to enable `Access-Control-Allow-Origin` for that URL

Thanks!

**Comment by vivekatoffice** [comment]
Related link:
[Default HTTP(S) Headers](https://help.axis.com/en-us/axis-os-knowledge-base#default-http-s-headers)
